products/storage/terraform.yaml

```diff
@@ -102,6 +102,37 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         ignore_read: true
   Object: !ruby/object:Overrides::Terraform::ResourceOverride
     exclude: true
+  HmacKey: !ruby/object:Overrides::Terraform::ResourceOverride
+    create_url: projects/{{project}}/hmacKeys?serviceAccountEmail={{service_account_email}}
+    self_link: projects/{{project}}/hmacKeys/{{access_id}}
+    id_format: "projects/{{project}}/hmacKeys/{{access_id}}"
+    import_format: ['projects/{{project}}/hmacKeys/{{access_id}}']
+    # This resource does not have a name field
+    skip_sweeper: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "storage_hmac_key"
+        primary_resource_id: "key"
+        vars:
+          account_id: "my-svc-acc"
+    docs: !ruby/object:Provider::Terraform::Docs
+      warning: |
+        All arguments including the `secret` value will be stored in the raw
```

Comment:
do we really not autogen this? interesting

Reply:
https://github.com/terraform-providers/terraform-provider-google/issues/5678